frontend-react/src/pages/Validate.tsx

```diff
@@ -6,16 +6,7 @@ import { AuthElement } from "../components/AuthElement";
 import { withCatch } from "../components/RSErrorBoundary";
 
 const Validate = () => {
-    return withCatch(
-        <FileHandler
-            headingText="ReportStream File Validator"
```

Comment:
Removed all of these props since the FileHandler component doesn't need to handle customized text for upload-to-send anymore.